mypy/message_registry.py

```diff
@@ -153,6 +153,10 @@ def with_additional_msg(self, info: str) -> ErrorMessage:
 FUNCTION_ALWAYS_TRUE: Final = ErrorMessage(
     "Function {} could always be true in boolean context", code=codes.TRUTHY_FUNCTION
 )
+ITERABLE_ALWAYS_TRUE: Final = ErrorMessage(
+    "{} which can always be true in boolean context. Consider using {} instead.",
```

Comment:
This message reads a bit weird to me but I guess it's consistent with the other ALWAYS_TRUE messages.

Reply:
On its own, I agree. It gets better once the variables are inserted. E.g.
```
"var" has type "Iterable[str]" which can always be true in boolean context.
Consider using "Collection[str]" instead.  [truthy-iterable]
```